docker-compose.yml

```
@@ -0,0 +1,251 @@
+---
+    version: '2.1'
+    
+    services:
+      controller:
+        image: ticdc:latest
+        build:
+          context: .
+          dockerfile: ./docker/Dockerfile
+        volumes:
+          - ./docker/data:/data
+          - ./docker/logs:/logs
+        command: -c "/usr/bin/tail -f /dev/null"
+        entrypoint: "/bin/sh"
+        depends_on:
+          - "upstream-tidb"
+          - "downstream-tidb"
+        restart: on-failure
+
+      capturer0:
+        image: ticdc:latest
+        build:
+          context: .
+          dockerfile: ./docker/Dockerfile
+        volumes:
+          - ./docker/data:/data
+          - ./docker/logs:/logs
+        entrypoint: "/cdc server"
+        command: 
+          - --pd-endpoints=http://upstream-pd:2379
```

```suggestion
          - --pd=http://upstream-pd:2379
```
We have unified  `--pd-addr` and `--pd-endpoints` to `--pd`, recommend to merge master and change this parameter. Ditto for the other two captures' command.